base/bitarray.jl

```diff
@@ -335,7 +335,7 @@ end
 @inline try_bool_conversion(x::Real) = x == 0 || x == 1 || throw(InexactError())
 @inline unchecked_bool_convert(x::Real) = x == 1
 
-function copy_to_bitarray_chunks!(Bc::Vector{UInt64}, pos_d::Int, C::Array{<:Real}, pos_s::Int, numbits::Int)
+function copy_to_bitarray_chunks!(Bc::Vector{UInt64}, pos_d::Int, C::AbstractArray{<:Real}, pos_s::Int, numbits::Int)
```

Comment:
Aha, sure, so then I'd keep this one restricted to `<:Real` and then also widen the method on 318 to `AbstractArray`.  That'd preserve the original intention of these two methods.